src/implementations/vanilla/components/accordion/accordion.js

```diff
@@ -152,7 +164,13 @@ export class Accordion {
       }
     }
 
-    return this;
+    this.toggles.forEach((item) =>
+      item.classList.remove('ecl-accordion__toggle--active'),
```

Comment:
i'd say no, because it's another thing, it's not about opened/closed accordion item, but the "latest" interaction by the user, regardless of the fact that the item is expanded or not.
Furthermore, we also need a way to distinguish between an action performed with the mouse and one performed with the keyboard, so...